tests/std/tests/GH_001017_discrete_distribution_out_of_range/test.cpp

```diff
@@ -6,6 +6,22 @@
 
 #include "bad_random_engine.hpp"
 
+template <class Distribution>
+void Test_for_NaN_Inf(Distribution&& distribution) {
+    for (bad_random_generator rng; !rng.has_cycled_through();) {
+        const auto rand_value = distribution(rng);
+        assert(!isnan(rand_value) && !isinf(rand_value));
+    }
+}
+
+template <class T>
+void Test_distributions() {

```

Comment:
Adding test coverage here is great, thanks. Although git history is always available, we conventionally add a comment like:
```cpp
// Additionally test GH-1174 "<random>: Some random number distributions could return NaN"
```
so if this section of the test fails, we can easily look up the bug/PR and understand the context.

Reply:
This is a relatively old PR. I didn't know this rule at that time. :)

Reply:
No problem at all! It's one of our unwritten conventions that I still need to collect in a document.